birdhouse/optional-components/scheduler-job-deploy_raven_testdata/config.yml.template

```diff
@@ -0,0 +1,14 @@
+- name: deploy_raven_testdata_to_thredds
```

Comment:
So for all the "deploy-data" job style, you are repeating the boilerplate config.yml ?  This is the reason that `deploy_data_job.env` exists: to generate the boilerplate with different schedule, name, command, and so on.

This is so that if the `deploy-data` script change, we have one single location to update the boilerplate template.

How about have one single boilerplate template in the `components/scheduler`, a new file next to `deploy_data_job.env` and use that boilerplate template to generate all `.yml` file for jobs that use `deploy-data` script?

Reply:
Let me think of something, I'll submit a PR on top of this PR.